app/controllers/solutions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class SolutionsController < ApplicationController
+  def create
+    @question = Question.find(params[:question_id])
+    @solution = @question.solutions.build(question_id: params[:question_id])
+    @solution.question_id = @question.id
```

Comment:
` @question.solutions.build`の書き方で，question_idはすでに渡せています。binding.pryで確認してみましょう。
ただ，回答で入力した値を受け取れていませんので，ストロングパラメータで受け取りましょう。

```suggestion
    @solution = @question.solutions.build(solution_params)
```

Reply:
- `app/models/solution.rb` に空文字投稿ができないようにバリデーション追加
- 入力した内容を、ストロングパラメーターで受け取る

以上を修正しました。